tests/conftest.py

```diff
@@ -38,3 +40,23 @@ def clock() -> Iterator[Clock]:
 @pytest.fixture
 def broker(clock: TestingClock) -> Iterator[LocalBroker[TPayload]]:
     yield LocalBroker(MemoryMessageStorage(), clock)
+
+
+@pytest.fixture(autouse=True)
+def run_migrations() -> Iterator[None]:
```

Comment:
Who is using this now ? This logic was in dataset_manager() before, but I don't see it being executed in the tests that were relying on dataset_manager.

Reply:
It's run before every test now regardless of whether they use dataset_manager (because of `autouse=True`)

Reply:
ah ok, nice